### PR TITLE
Task

### PR DESCRIPTION
**Pull request by kimberleyworld**
Please could you review the nav bar.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+	<head>
+		<title>Kimberley Dobney Portfolio</title>
+		<link type="text/css" rel="stylesheet" href="normalize.css">
+		<link type="text/css" rel="stylesheet" href="styles.css">
+	</head>
+	<body>
+		<nav>
+			<h4>Kimberley Dobney Portfolio</h4>
+				<div class="pages">
```
Comment (lilyboughton) [inline index.html]: This whole <div> doesn't need to be indented. It's at the same level as your <h4>

**Review by lilyboughton — CHANGES_REQUESTED** [review]
Your code is beautifully neat, well-organised and does what it's supposed to do. It might be worth considering using some @media queries to adjust how the navbar displays on a mobile.

**Review by lilyboughton — APPROVED** [review]
Looks great, nice one :)

**Comment by Reductito** [comment]
Header looks good nothing wrong from what I can see

**Pull request merged**